### PR TITLE
new blog of mysql-on-k8s

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao@kubesphere.io>

### CODE REVIEW

File: content/zh/blogs/mysql-on-k8s.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+规模越大，出现误操作的概率会越高，严重的甚至要 **“从删库跑路”**。误删除关键数据，对于一般企业而言，应对能力较弱，危害很可能是致命的。
+
+### 4、资源弹性
+
+传统物理机部署不具备秒级弹性的能力，来针对 MySQL 在高峰或低谷时资源的自动弹性伸缩。例如，在业务高峰扩展 CPU、内存等资源，在低峰期收回闲置资源。
+
+如果设计一次电商秒杀的场景持续时间是 30 分钟，那么我们可以在 30 分钟内，将网络、CPU、内存、磁盘等资源提升到最大。在秒杀结束之后，释放这些资源，极大节省成本。
+
+### 主流解决方案
+
```
Comment (yangchuansheng) [inline content/zh/blogs/mysql-on-k8s.md:44]: Replace Heading level 3 with Heading level 2

Comment (faweizhao26) [reply content/zh/blogs/mysql-on-k8s.md]: Done.

**Pull request merged**